### ISSUE TITLE
Build fails  to replace cssVars

### ISSUE THREAD

**Issue by difelice** [facebook/fixed-data-table#112]
When building via `npm run build-dist`, *cssVars* are not replaced properly.
```
> NODE_ENV=production webpack && NODE_ENV=production COMPRESS=1 webpack

<css input>:27:3: variable 'scrollbar-size' is undefined and used without a fallback
<css input>:33:3: variable 'scrollbar-track-color' is undefined and used without a fallback
<css input>:34:3: variable 'scrollbar-size-large' is undefined and used without a fallback
...
```

It works by prepending "--" to variables: https://github.com/difelice/fixed-data-table/commit/13f80a62c5d9c8913963e401df5c85f5c457a218

**Comment by pieterv** [comment]
I fixed this with #110, if you pull down master and then do a fresh `npm install` it should work :)

**Comment by difelice** [comment]
Thank you so much!
